test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -107,6 +107,12 @@ namespace integration_framework {
     return *this;
   }
 
+  IntegrationTestFramework &IntegrationTestFramework::sendQuery(
+      const iroha::model::Query &qry) {
+    sendQuery(qry, [](const auto &) {});
+    return *this;
+  }
```

Comment:
Cover with test.

Reply:
I really hope these messages are not automated. In this case a simple "Cover `sendQuery(qry)` method" message would have been enough.